docs/platform/release-notes/tizen-7-0-m1.md

```diff
@@ -0,0 +1,545 @@
+# Tizen 7.0 Public M1
+
+Release date: May 31, 2022
+
+
+## Release Notes
+
+### System (Kernel and System framework)
+
+#### New and changed features
+
+- Kernel and U-Boot upgrade
+  - Kernel for Raspberry Pi 4 has been upgraded to version 5.10.95.
+  - U-boot has been upgraded to v2021.10.
+- PREEMPT_RT Support
+  - PREEMPT_RT applied kernel for Raspberry Pi 4 has been supported.
+- System and resource management
+  - Cgroup hierarchy and configurations have been improved.
+  - Partition detection mechanism in initrd has been optimized.
+- Device and sensor management
+  - Sensor attribute set API set for lidar sensor has been developed.

```

Comment:
"Sensor attribute set API set" I am confused as to what this means. Could you please clarify? @wtprince

Reply:
Sensor attribute set APIs for lidar sensor have been developed.
This is right.

Reply:
```suggestion
  - Sensor attribute set APIs for lidar sensor have been developed.

```

Reply:
Will accept.

Reply:
FYI : Global : About "APIs"

"APIs" were requested to be changed to "API set" some years before.
So we changed "APIs" to "API Set" and keeping the consistency in release notes.

**Reference :** https://github.com/Samsung/tizen-docs/pull/342